src/Domain/ContentRenderer/CodeRenderer.php

```diff
@@ -74,16 +74,11 @@ private function getWrapperAttributes( string $contentUrl ): array {
 	/**
 	 * @return string
```

Comment:
This is no longer necessary and can be removed.

Reply:
Doing that in https://github.com/ProfessionalWiki/ExternalContent/pull/46. This PR is only for the CSS change.

Reply:
There's a conflict now because this PR also contains the same change.

Reply:
Fixed and noted.